kong/runloop/certificate.lua

```diff
@@ -217,6 +218,13 @@ local function find_certificate(sni)
     return default_cert_and_key
   end
 
+  local res, err = typedefs.wildcard_host.custom_validator(sni)
+  if not res then
+    log(DEBUG, "invalid SNI '" .. sni .. "', " .. err ..
```

Comment:
Please use `kong.log.debug` instead; also, note that the concats aren't needed (you can pass in multiple args).